whist/server/util.py

```diff
@@ -0,0 +1,19 @@
+"""Multi Thread adapter.."""
+import asyncio
+
+
+# pylint: disable=too-few-public-methods
+class ThreadManager:
+    """
+    Handles multi threads calls.
+    """
+
+    @staticmethod
+    def run(func, args) -> None:
+        """
+        Runs an async function in a new thread.
+        :param func: to be called
+        :param args: to be parsed to the call.
+        :return: None
+        """
+        asyncio.run(func(args))
```

Comment:
```suggestion
        asyncio.create_task(func(args))
```

Reply:
Make sure to create exactly _ONE_ event loop before hand!

Reply:
maybe make more functions async as well?

Reply:
fastapi uses https://anyio.readthedocs.io/en/stable/ internally, so we should probably use that as well?